### DISCUSSION TITLE
Auto table Alias option

### DISCUSSION THREAD

**Discussion by agentwalker** [microsoft/azuredatastudio#21649] [Ideas]
There are a couple of plugins that already do this for SSMS but I can't seem to find anything that does this currently for Data Studio.

for example.

SELECT * FROM MyAddressLocation

when you tab on the 'MyAddressLocation' it should do the following
SELECT * FROM MyAddressLocation mal

then for the joins 
SELECT * FROM MyAddressLocation mal
  INNER JOIN ShopsInLocation ail on ail.postcode = mal.postcode

any chance guys 

thanks for the great app.

**Comment by erinstellato-ms** [comment]
Closing as I see you've entered an [issue](https://github.com/microsoft/azuredatastudio/issues/24149) (which was going to be my suggestion).  Thanks!
